app/main.py

```diff
@@ -1,3 +1,52 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | float) -> Distance:
```

Comment:
the argument can be Distance, float or int

Reply:
not fixed

Reply:
Not fixed

Reply:
Not fixed